pymeasure/adapters/prologix.py

```diff
@@ -200,7 +200,7 @@ def ask(self, command):
     def write(self, command, **kwargs):
         """Write a string command to the instrument appending `write_termination`.
 
-        If the GPIB address in :attr:`.address` is defined, it is sent first.
+        If the GPIB address in :attr:`self.address` is defined, it is sent first.
```

Comment:
Why this change?

The "attr" indicates an attribute of the class/instance. Maybe the period is superfluous.

Reply:
This change was triggered by the following sphinx error in the first CI run:
`/pymeasure/pymeasure/pymeasure/adapters/prologix.py:docstring of pymeasure.adapters.prologix.PrologixAdapter.write:3: WARNING: more than one target found for cross-reference 'address': pymeasure.instruments.anaheimautomation.DPSeriesMotorController.address, pymeasure.instruments.tdk.tdk_gen40_38.TDK_Gen40_38.address, pymeasure.instruments.tdk.tdk_gen80_65.TDK_Gen80_65.address`

I don't know why it was triggered now since the `DPSeriesMotorController` has been around for a while. Removing the period is best, thanks for the suggestion.